src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.io.*;
+
+import java.util.ArrayList;
+
+
+/**
+ * Handles the saving and loading of tasks to and from a file in a directory.
+ */
+public class Storage {
+    java.nio.file.Path path = java.nio.file.Paths.get( "./","src", "main", "data", "duke.txt");
```

Comment:
Perhaps we could replace the fully qualified path with an import statement instead?